packages/app-extensions/src/formField/formattedValueFactory.js

```diff
@@ -17,7 +17,19 @@ const getOptions = (type, modelField, formData) => {
   return {}
 }
 
+const typeMapping = {
```

Comment:
maybe we can find a  better place for this mapping. i will look into it and give you a more detailed feedback